src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -92,16 +92,74 @@
 {% macro form_widget_with_error(form, vars) %}
   {% set vars = vars|default({}) %}
   {% set attr = vars.attr|default({}) %}
-  {% set errors = form.vars.errors %}
+  {% set errors = [] %}
 
-  {% set attr = attr|merge({'class': (attr.class is defined ? attr.class : '') ~ (errors|length > 0 ? ' is-invalid' : '')} ) %}
+  {% if not form.vars.valid  %}
+    {% for parentError in form.vars.errors %}
+      {% set errors = errors|merge([parentError.message]) %}
+    {% endfor %}
+
+    {% for child in form.children %}
+      {% for error in child.vars.errors %}
+        {% set errors = errors|merge([error.message]) %}
+      {% endfor %}
+    {% endfor %}
+  {% endif %}
+
+  {% set attr = attr|merge({'class': (attr.class is defined ? attr.class : '')} ) %}
   {% set vars = vars|merge({'attr': attr}) %}
 
   {{ form_widget(form, vars) }}
+  {% if errors|length > 0 %}
+
+    {# form form types which has locales and there are more then 1 error , additional errors are displaying inside popup #}
+    {% set errorPopover = null %}
+    {% if form.vars.locales is defined and
+          form.vars.locales is not null and
+          form.vars.locales|length > 1 and
+          errors|length > 1
+    %}
+      {% set popoverErrors = errors|slice(1) %}
+      {% set errors = [errors|first] %}
+      {% set popoverContainer = 'locale-input-error-popover-container-'~form.vars.id %}
+
+      <div class="{{ popoverContainer }}"></div>
+
+      {% set popoverErrorContent %}
+        <ul class="popover-error-list">
+          {% for popoverError in popoverErrors %}
+              <li>
+                <span class="invalid-feedback">{{ popoverError }}</span>
+              </li>
+          {% endfor %}
+        </ul>
+      {% endset %}
+
+      {% set errorPopover %}
+        <span
+          class="js-locale-input-error-popover-trigger"
+          data-toggle="locale-input-popover"
+          data-placement="top"
+          data-html="true"
+          data-template='<div class="popover js-locale-input-error-popover" role="tooltip"><h3 class="popover-header"></h3><div class="popover-body"></div></div>'
+          data-content='{{ popoverErrorContent }}'
+          data-trigger="hover"
+          data-container=".{{ popoverContainer }}"
+        >
+          {#todo: ask about translation domain#}
+          <u>+ {{ '%count% more'|trans({'%count%': popoverErrors|length}, 'Admin.Actions') }}</u>
```

Comment:
Hi @LouiseBonnard 

due to some UX update 
![err](https://user-images.githubusercontent.com/17051250/50739652-11cb4900-11ec-11e9-9d08-3266e7e00322.png)

I included new translation : `%count% more` under `Admin.Actions` - is it good?

Also I this case I did not included `+` sign in the translation - should it be included as well?

Reply:
Hi there, @tomas862! Okay, thank you, can you tell me what is it supposed to count?

Reply:
Hi @LouiseBonnard  - yes , it is supposed to count remaining erros which appears under multi-language field.

for instance,

We have 4 errors displayed under multilanguage input and it will be displayed like this:

"My error"  + `{{ '%count% more'|trans({'%counte%' : 3}, 'Admin.Actions') }}`. In the pop-up it will be displayed remaining 3 errors. You can see how it looks in the image above :slightly_smiling_face:

Reply:
Alright, then I would rather localize it in _Admin.Global_.